app/process/watcher/transfer/watcher.go

```diff
@@ -156,6 +162,7 @@ func (ctw Watcher) processTransaction(tx mirror_node.Transaction, q *pair.Queue)
 		return
 	}
 
-	transferMessage := transfer.New(tx.TransactionID, ethAddress, nativeAsset, wrappedAsset, amount, ctw.contractService.Address().String())
-	q.Push(&pair.Message{Payload: transferMessage})
+	mockChainID := big.NewInt(0)
+	transferMessage := transfer.New(tx.TransactionID, ethAddress, nativeAsset, wrappedAsset, amount, ctw.contractServices[mockChainID].Address().String())
```

Comment:
Shouldn't this be the `target chain ID` instead of the `source chain ID`?

Reply:
The logic here is simply mocked for the purpose of finishing the EVM -> Hedera flow. Next PR will cover these mocked structures.